mobile/status.go

```diff
@@ -604,3 +604,14 @@ func ExportNodeLogs() string {
 	}
 	return string(data)
 }
+
+// ChaosModeSetUpstreamURL changes the URL of the upstream RPC client.
+func ChaosModeSetUpstreamURL(url string) string {
```

Comment:
I would pass a Boolean here and the just check the urls of private and public RPC endpoints.